packages/dashboard-core-plugins/src/linker/LinkerOverlayContent.tsx

```diff
@@ -109,13 +124,37 @@ export class LinkerOverlayContent extends Component<
     return LayoutUtils.getTabPoint((glContainer as unknown) as Container);
   }
 
+  handleOperatorChanged(linkId: string, type: FilterTypeValue): void {
+    const { links, onLinksUpdated } = this.props;
+    const newLinks: Link[] = links.map(link =>
+      link.id === linkId ? { ...link, operator: type } : link
+    );
+    onLinksUpdated(newLinks);
+  }
+
   handleMouseMove(event: MouseEvent): void {
     this.setState({
       mouseX: event.clientX,
```

Comment:
I think you should set the `mode` here as well based on if the `altKey` is pressed... Or listen for when the window goes out of focus and change the mode back.
Reason being when I alt-tab out of this window, the window never gets the AltUp, so when I come back, it still looks like it's going to delete it.